.github/workflows/ci_standalone.yml

```diff
@@ -24,15 +24,28 @@ jobs:
         with:
           python-version: "3.10"
 
+      - name: Set environment variables
+        run: |
+          echo "OCI_ENV_PATH=${HOME}/work/galaxy_ng/oci_env" >> $GITHUB_ENV
+
       - name: Update apt
         run: sudo apt -y update
 
       - name: Install LDAP requirements
-        run: sudo apt-get install -y libsasl2-dev python-dev libldap2-dev libssl-dev build-essential
+        run: sudo apt-get install -y libsasl2-dev python-dev libldap2-dev libssl-dev build-essential tree
 
       - name: Install docker-compose
         run: pip3 install --upgrade docker-compose
 
+      - name: setup insights test instance
+        run: |
+          git clone https://github.com/newswangerd/oci_env.git --branch feature/db-snapshot $OCI_ENV_PATH
+          pip install -e $OCI_ENV_PATH/client/
+          mkdir $OCI_ENV_PATH/db_back/
+          cp dev/sync_tests/fixture.tar.gz $OCI_ENV_PATH/db_back/fixture.tar.gz
+          oci-env -e dev/sync_tests/sync-test.compose.env compose build
+          oci-env -e dev/sync_tests/sync-test.compose.env compose up
```

Comment:
Does this detach? I assumed with `-d` it would run forever and block any further tasks in the workflow.

Reply:
@jctanner I'm still debugging the CI actions, so this file is actively being modified. Sorry, I didn't think it would ping you for reviews until I marked it as "not draft"